### DISCUSSION TITLE
Glowroot central without Cassandra and Docker

### DISCUSSION THREAD

**Discussion by pcouas** [glowroot/glowroot#1031] [Q&A] [answered]
Hi,
Could i use glowroot-central just for monitoring an webapp application on Linux/Tomcat and also Spring batch 4 on another Linux server without Cassandra and docker ? This is same application in two part on two servers ?
I could not install Product on Customer Computer server

Regards 
Philippe

**Comment by Never-lab** [comment, marked as answer]
Short answer: **Central needs a standalone Cassandra**. Docker is optional; Cassandra is not. There is no "Central without C*" mode.

If you cannot install Cassandra on the customer box, use **embedded** instead: `-javaagent` on each JVM (Tomcat + Spring Batch) with unique `agent.id` / `multi.dir` as needed — each JVM gets its own local UI/storage, not one shared Central UI.

See [Choosing Embedded vs Central](https://github.com/glowroot/glowroot/wiki/Choosing-Embedded-vs-Central) and [Central Collector Installation](https://github.com/glowroot/glowroot/wiki/Central-Collector-Installation).
